src/asset-swapper/utils/market_operation_utils/sampler_operations.ts

```diff
@@ -1947,14 +1948,18 @@ export class SamplerOperations {
                                     ERC20BridgeSource.Balancer,
                                 ),
                             );
+                    case ERC20BridgeSource.Beethovenx:
```

Comment:
I think the same OP token check needs to be added here.

Reply:
ack